lib/ui/view/edit_profile/edit_profile_viewmodel.dart

```diff
@@ -1,26 +1,24 @@
+import 'package:hng/app/app.locator.dart';
+import 'package:hng/models/profile_model.dart';
+
+import 'package:hng/package/base/server-request/api/http_api.dart';
+import 'package:hng/services/connectivity_service.dart';
+import 'package:hng/services/current_user_profile.dart';
+import 'package:hng/services/local_storage_services.dart';
+import 'package:hng/utilities/enums.dart';
+import 'package:hng/utilities/storage_keys.dart';
 import 'package:stacked/stacked.dart';
 import 'package:stacked_services/stacked_services.dart';
 
-import '../../../app/app.locator.dart';
-import '../../../general_widgets/app_toast.dart';
-import '../../../models/profile_model.dart';
-import '../../../package/base/server-request/api/http_api.dart';
-import '../../../services/connectivity_service.dart';
-import '../../../services/local_storage_services.dart';
-import '../../../utilities/enums.dart';
-import '../../../utilities/storage_keys.dart';
-
-ProfileModel userData = new ProfileModel();
-String _nameD = '', _displayD = '', _statusD = '', _phoneD = '';
 String _name = '', _display = '', _status = '', _phone = '';
 
-class EditProfileViewModel extends BaseViewModel {
+class EditProfileViewModel extends FutureViewModel {
+  ProfileModel userData = ProfileModel();
   final snackbar = locator<SnackbarService>();
   final navigationService = locator<NavigationService>();
   final connectivityService = locator<ConnectivityService>();
   final _api = HttpApiService('https://api.zuri.chat/');
```

Comment:
Use the api service from api_service.dart

Reply:
Ok